woocommerce/payment-gateway/apple-pay/class-sv-wc-payment-gateway-apple-pay-frontend.php

```diff
@@ -183,6 +183,44 @@ public function render_button() {
 	}
 
 
+	/**
+	 * Renders a notice informing the customer that by purchasing they are accepting the website's terms and conditions.
+	 *
+	 * Only displayed if a Terms and conditions page is configured.
+	 *
+	 * @internal
+	 *
+	 * @since 5.5.4
+	 */
+	public function render_terms_notice() {
+
+		if ( apply_filters( 'woocommerce_checkout_show_terms', true ) && function_exists( 'wc_terms_and_conditions_checkbox_enabled' ) && wc_terms_and_conditions_checkbox_enabled() ) {
```

Comment:
we can document this by pointing to the [hook's original location](https://github.com/skyverge/wc-plugins/wiki/PHP-Best-Practices#documenting-actions--filters)